spec/advanced/hashTable_test.js

```diff
@@ -32,14 +32,79 @@ describe('HashTable', () => {
 	})
 
 	context('put(key)', () => {
-		it('returns the hashed integer value of a given string', () => {
+		it('adds a new key-value pair to the hash table', () => {
 			const ht = new HashTable()
 			ht.put('Enumerations', 'Flexible')
 			ht.put('Swift', 'Language')
-			// console.log(ht);
 			expect(ht.size() ).to.equal(2)
 			expect(ht.myGet('Enumerations') ).to.equal('Flexible')
 		})
 	})
 
+	context('myGet(key)', () => {
+		it('returns the value associated with a key', () => {
+			const ht = new HashTable()
+			ht.put('Enumerations', 'Flexible')
+			ht.put('Swift', 'Language')
+			expect(ht.size() ).to.equal(2)
+			expect(ht.myGet('Swift') ).to.equal('Language')
+		})
+	})
+
+	context('myGet(key)', () => {
+		it('returns -1 if the key does not exist in the table', () => {
+			const ht = new HashTable()
+			ht.put('Enumerations', 'Flexible')
+			ht.put('Swift', 'Language')
+			expect(ht.size() ).to.equal(2)
+			expect(ht.myGet('Classes and Structures') ).to.equal(-1)
+		})
+	})
+
+	context('myRemove(key)', () => {
+		it('returns false if the key does not exist in the table', () => {
+			const ht = new HashTable()
+			ht.put('Enumerations', 'Flexible')
+			ht.put('Swift', 'Language')
+			// expect(ht.size() ).to.equal(2)
+			ht.myRemove('Enumerations')
+			expect(ht.myContains('Enumerations') ).to.equal(false)
+		})
+	})
+
+	context('contains(key)', () => {
+		it('returns true is the table contains a given key', () => {
+			const ht = new HashTable()
+			ht.put('Enumerations', 'Flexible')
+			expect(ht.size() ).to.equal(1)
+			expect(ht.myContains('Enumerations') ).to.equal(true)
+		})
+		it('returns false is the table contains a given key', () => {
```

Comment:
Lines 76 and 82 have the same message, not crucial just a little confusing at first.